props/saber_fett263_buttons.h

```diff
@@ -4209,14 +4211,13 @@ SaberFett263Buttons() : PropBase() {}
         if (swing_blast_) {
           SaberBase::DoBlast();
           return true;
-        }
-        if (check_blast_ && battle_mode_) {
-          if (!swing_blast_ && millis() - last_blast_millis_ > 2000) {
-            swing_blast_ = true;
-            hybrid_font.PlayCommon(&SFX_blstbgn);
-            SaberBase::DoBlast();
+        } else {
```

Comment:
There is really no need for an else here since the return on the previous line prevents the processor from going here anyways, even without the else.

If you do want the else, then you might want to remove the "return true" on the previous line since it is now redundant. The "return true" on line 4223 can do the job instead.